.github/workflows/run_unit_tests.yml

```diff
@@ -29,18 +29,19 @@ jobs:
     - name: Set up Python ${{ matrix.python-version }}
       uses: actions/setup-python@v2
       with:
-        python-version: ${{ matrix.python-version }}
-    - name: Install dependencies
+        python-version: ${{ matrix.python-version }} 
+    - name: Install uv
```

Comment:
Let use the official uv Github Action to install uv, see https://github.com/openradx/adit-radis-shared/blob/f176fa95f8e0619d1c48707b38edc74439a83796/.github/workflows/ci.yml#L16